Resources/Prototypes/_Floof/psionics.yml

```diff
@@ -25,3 +25,23 @@
   - !type:RemovePsionicStatSources
   - !type:RemoveAssayFeedback
     assayFeedback: hypnosis-power-feedback
+
+- type: psionicPower
+  id: GlimmerLeak
+  name: GlimmerLeak
+  powerCategories:
+    - Passive
+    - Dangerous
+  powerSlotCost: 0
+  initializeFunctions:
+    - !type:AddPsionicPowerComponents
+      components:
+      - type: GlimmerSource
+    - !type:AddPsionicAssayFeedback
+      assayFeedback: glimmer-leak-power-feedback
+  removalFunctions:
+    - !type:RemovePsionicPowerComponents
+      components:
+      - type: GlimmerSource
```

Comment:
I'm not sure how balanced this trait would be in the first place, but if we are adding it, it should produce less glimmer than a prober (maybe like 30% of the normal prober rate? we don't know how many people would decide to take this trait and lock up in dorms)